actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java

```diff
@@ -0,0 +1,64 @@
+package org.sunbird.user.actors;
+
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.*;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ActorConfig(
+        tasks = {"validateCertificate"},
+        asyncTasks = {}
+)
+public class CertificateActor extends UserBaseActor{
```

Comment:
Java doc for all class and public method.

Reply:
Done